scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -137,9 +137,10 @@ workflow GvsCreateFilterSet {
         gatk_docker = "us.gcr.io/broad-gatk/gatk:4.3.0.0",
         extract_interval_list = interval_list,
         score_interval_list = interval_list,
-        snp_annotations = "-A AS_QD -A AS_MQRankSum -A AS_ReadPosRankSum -A AS_FS -A AS_MQ -A AS_SOR",
-        indel_annotations = "-A AS_FS -A AS_ReadPosRankSum -A AS_MQRankSum -A AS_QD -A AS_SOR",
+        snp_annotations   = "-A AS_QD -A AS_MQRankSum -A AS_ReadPosRankSum -A AS_FS -A AS_MQ -A AS_SOR",
+        indel_annotations = "-A AS_QD -A AS_MQRankSum -A AS_ReadPosRankSum -A AS_FS -A AS_MQ -A AS_SOR",
         use_allele_specific_annotations = true,
+        monitoring_script = "gs://gvs_quickstart_storage/cromwell_monitoring_script.sh"
```

Comment:
is this going to work with the Cromwell Local backend test?

Reply:
Do you mean our integration test? I modified it to be a parameter to JointVcfFiltering.wdl so that that wdl's cromwell test will work. But now I realize that this might also break our integration tests?